### ISSUE TITLE
Multi-elision appearing on experiment but not prod

### ISSUE THREAD

**Issue by abziegler** [harvard-lil/h2o#839]
Not sure what to make of this annotation inconsistency, but experiment renders this as three separate elisions and prod renders as a single elision:

experiment - see para 117-120 - https://h2o-experiment.lil.tools/casebooks/79342-corporations/resources/2.2.3.5-in-re-trulia-inc-stockholder-litigation-del-ch-2016#/

![image](https://user-images.githubusercontent.com/2140548/62072050-cc17b180-b20b-11e9-96be-44b729edb65e.png)

prod - see para 117-120 - https://opencasebook.org/casebooks/79342-corporations/resources/2.2.3.5-in-re-trulia-inc-stockholder-litigation-del-ch-2016#/

![image](https://user-images.githubusercontent.com/2140548/62072089-df2a8180-b20b-11e9-81fd-4b70890eb932.png)

**Comment by rebeccacremona** [comment]
Ooh, this is a fun one. Whitespace! We've got lots of extra `<br`s happening, not sure why.

Prod:
![image](https://user-images.githubusercontent.com/11020492/62163285-0ce4f880-b2e8-11e9-8a1c-06f7b76df22a.png)

Experiment:
![image](https://user-images.githubusercontent.com/11020492/62163359-2dad4e00-b2e8-11e9-9a5c-060a4214b946.png)
```
# resource.raw_content.content
<p><strong>IT IS SO ORDERED.</strong><br /><br /></p>\n<p>[1] By closing, the transaction value had fallen to $2.5 billion

# resource.content
<p><strong>IT IS SO ORDERED.</strong><br>\n<br></p><p>[1] By closing, the transaction value had fallen to $2.5 billion
```

**Comment by holtchesley** [comment]
This particular issue seems resolved due to #1075, but the fix therein needs to be adapted to work for replacement annotations as well to catch an edge-case shown in pp92-113 of https://h2o-experiment.lil.tools/casebooks/79342-corporations/resources/2.2.3.5-in-re-trulia-inc-stockholder-litigation-del-ch-2016#/
